modules/agent-configuration-parameters.adoc

```diff
@@ -0,0 +1,269 @@
+
+These values are specified in the `agent-config.yaml` file.
+
+[NOTE]
+====
+After installation, you cannot modify these parameters in the `agent-config.yaml` file.
+====
+
+[id="agent-configuration-parameters-required_{context}"]
+== Required configuration parameters
+
+Required Agent configuration parameters are described in the following table:
+
+.Required parameters
+[cols=".^2l,.^3,.^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|apiVersion:
+|The API version for the `agent-config.yaml` content.
+The current version is v1.
+The installation program may also support older API versions.
+|String
+
+|kind:
+|(needs info)
+|(needs info)
+
+|metadata:
+|Kubernetes resource `ObjectMeta`, from which only the `name` parameter is consumed.
+|Object
+
+|metadata:
+  name:
+|The name of the cluster.
+DNS records for the cluster are all subdomains of `{{.metadata.name}}.{{.baseDomain}}`.
+|String of lowercase letters and hyphens (`-`), such as `dev`.
+|====
+
+
+[id="agent-configuration-parameters-optional_{context}"]
+== Optional configuration parameters
+
+Optional Agent configuration parameters are described in the following table:
+
+.Optional parameters
+[cols=".^2l,.^3,.^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|rendezvousIP:
+|The IP address of the node that performs the bootstrapping process as well as running the `assisted-service` component.
+You must provide the rendezvous IP address when you do not specify at least one host's IP address in the `networkConfig` parameter.
+If this address is not provided, one IP address is selected from the provided hosts' `networkConfig`.
+|IPv4 or IPv6 address.
+IPv6 is supported only on bare metal platforms.
+
+|bootArtifactsBaseURL:
+|(needs info)
+|(needs info)
+
+|additionalNTPSources:
+|(needs info)
+|(needs info)
+
+|hosts:
+|Host configuration.
+The number of hosts defined must not exceed the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters.
+|An array of host configuration objects.
+
+|hosts:
+  hostname:
+|Host name.
+Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+Each host must have a unique hostname supplied by one of these methods.
+|(needs info)
+
+|hosts:
+  interfaces:
+|(needs info)
+|(needs info)
+
+|hosts:
+  interfaces:
+    name:
+|(needs info)
+|(needs info)
+
+|hosts:
+  interfaces:
+    macAddress:
+|(needs info)
+|(needs info)
+
+|hosts:
+  rootDeviceHints:
+|Enables provisioning of the {op-system-first} image to a particular device.
+It examines the devices in the order it discovers them, and compares the discovered values with the hint values.
+It uses the first discovered device that matches the hint value.
+|(needs info)
+
+|hosts:
+  rootDeviceHints:
+    deviceName:
+|(needs info)
+|(needs info)
+
+|hosts:
+  networkConfig:
```

Comment:
The networkConfig field is the host network definition. It must match the Network Management API defined in https://nmstate.io/.